cmd/examples/server.cpp

```diff
@@ -614,7 +618,7 @@ class MyServer : public quicr::Server
         std::optional<quicr::Extensions> extensions;
         quicr::Bytes data;
     };
-    std::map<quicr::TrackNamespaceHash, std::multimap<quicr::messages::GroupId, CacheObject>> cache_;
+    std::map<quicr::TrackNamespaceHash, quicr::Cache<quicr::messages::GroupId, std::vector<CacheObject>>> cache_;
```

Comment:
If we use set, it could ensure order based on object_id. Also, it would prevent duplicates, which is nice to dedup.

Reply:
Ah, good idea, I'll add that! Thanks!